src/histolab/filters/image_filters.py

```diff
@@ -698,13 +698,20 @@ class RagThreshold(ImageFilter):
         Color proximity versus space proximity factor. Default is 10.0
     threshold : int, optional
         Threshold value for combining regions. Default is 9.
+    return_labels : bool, optional
+        If True, returns a labeled array where the value denotes segment
+        membership. Otherwise, returns a PIL image where each segment is colored
+        by the average color in it. Default is False.
 
     Returns
     -------
-    PIL.Image.Image
+    PIL.Image.Image, if not ``return_labels``
         Each segment has been colored based on the average
         color for that segment (and similar segments have been combined).
 
+    np.ndarray, if ``return_labels``
+        Value denotes segment membership.
+
```

Comment:
I'm concerned on the multiple return type of this filter. This can be a problem when will be the moment of using any static type checker like `mypy` or similar. Maybe a solution could be split this filter into 2 filters? I don't know it's just a sort of rubberduck thinking 🤷🏽‍♂️ . Let me know what do you think @kheffah @alessiamarcolini